app/components/bulk_upload_error_row_component.rb

```diff
@@ -18,7 +18,7 @@ def tenant_code
   def tenant_code_html
     return if tenant_code.blank?
 
-    content_tag :span, class: "govuk-!-margin-left-3" do
+    content_tag :span, class: "govuk-!-margin-left-3", style: "font-weight: 400;" do
```

Comment:
Isn't that the default font weight?

Reply:
This tag is wrapped around in a `h3` so although the text before it is wrapped in `<strong>` it is redundant as the whole line was having the font-weight = 700 applied as a result of the `h3`.

`<h3 class="govuk-summary-card__title"><strong>Row <%= row %></strong> <%= tenant_code_html %> <%= property_ref_html %></h3>`

I should probably remove the `<strong>` tags or change it from a `<h3>`

Reply:
I'll add `govuk-!-font-weight-regular` class to the `h3`, and remove the inline styling from all 3 tags I added it to here.